src/widgets/tdarr/HealthCheckStatus.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  ActionIcon,
+  Divider,
+  Group,
+  HoverCard,
+  Indicator,
+  MantineColor,
+  RingProgress,
+  Stack,
+  Text,
+} from '@mantine/core';
+import { IconHeartbeat } from '@tabler/icons-react';
+import { useTranslation } from 'next-i18next';
+import { useColorScheme } from '~/hooks/use-colorscheme';
+import { TdarrStatistics } from '~/server/api/routers/tdarr';
+
+interface StatisticsBadgeProps {
+  statistics?: TdarrStatistics;
+}
+
+export function HealthCheckStatus(props: StatisticsBadgeProps) {
+  const { statistics } = props;
+
+  const { colorScheme } = useColorScheme();
+  const { t } = useTranslation('modules/tdarr-queue');
+
+  if (!statistics) {
+    return (
+      <ActionIcon loading>
+        <IconHeartbeat size={20} />
+      </ActionIcon>
+    );
+  }
+
+  const indicatorColor = statistics.failedHealthCheckCount
+    ? 'red'
+    : statistics.stagedHealthCheckCount
+      ? 'yellow'
+      : 'green';
+
+  return (
+    <HoverCard position="bottom" width={250} shadow="sm" openDelay={0} closeDelay={0}>
+      <HoverCard.Target>
+        <ActionIcon
+          variant="subtle"
+          style={{
+            cursor: 'initial',
+          }}
+        >
+          <Indicator
+            color={textColor(indicatorColor, colorScheme)}
+            size={8}
+            processing={!!statistics.stagedHealthCheckCount}
```

Comment:
It's an accessibility setting for people that get stressed out when they see that it's flashing, you can search in the code to see how the value is gotten and used

Reply:
IMO, we shouldn't use the `disablePingPulse` setting here since it's not a ping indicator. I'll just remove the pulsing since it doesn't really add any information anyways.